src/include/compat.h

```diff
@@ -76,6 +76,11 @@
 #endif
 #endif
 
+/* detected on Alpine platform, might require  <sys/stat.h> */
```

Comment:
please remove this comment. as it's misleading. the reason `ACCESSPERM` is `#defined`ed here is that musl libc does not defined it as it is not required by POSIX or C11 standards at the time of writing.